trick_source/data_products/Log/MatLab4.cpp

```diff
@@ -364,6 +364,9 @@ int MatLab4LocateParam( char *file_name , char *param_name , char *time_name ) {
                         case 5:
                                 size = 1 ;
                                 break ;
+                        default :
+                                std::cerr << "Matlab4 " << __FUNCTION__ << "ERROR: invalid matrix size" << std::endl;
+                                size = -1; 
```

Comment:
Do you want a `break` here for consistency?

```suggestion
                                size = -1 ;
                                break ;
```

Reply:
fixed, thanks